guardian/src/main/java/org/rfcx/guardian/guardian/audio/capture/AudioCaptureWavRecorder.java

```diff
@@ -304,7 +304,7 @@ public void startRecorder() {
 
 			captureFilePayloadSizeInBytes = 0;
 			audioRecorder.startRecording();
-			audioRecorder.read(uncompressedOutputBuffer, 0, uncompressedOutputBuffer.length);
+//			audioRecorder.read(uncompressedOutputBuffer, 0, uncompressedOutputBuffer.length);
```

Comment:
Also delete. :)

Reply:
Was this causing the bug? Or was it related to wake lock?

Reply:
> Was this causing the bug? Or was it related to wake lock?

This cause the bug, it was from Topher's first code.
For the wake lock I added later and test it but still error.
After that I comment this calling and it works.
So Im not sure if it just comment this line that solve the bug or both with wake lock and comment.